### DISCUSSION TITLE
好像又寄了

### DISCUSSION THREAD

**Discussion by xhqss** [LmeSzinc/AzurLaneAutoScript#2457] [Q&A]
没法更新
![屏幕截图 2023-04-06 123359](https://user-images.githubusercontent.com/112264454/230273050-f64360f8-6ca1-47fc-8532-3c250c670e0f.png)

**Comment by KokoronoSikima** [comment]
![image](https://user-images.githubusercontent.com/67483626/230394342-ad568b2d-3231-4f9d-94a9-9be70868e73b.png)
我的也是。

**Reply by jarviexu** [reply]
你们现在能用吗？重装当晚可以用，第二天开机又不行了
![Uploading 微信图片_20230603194044.png…]()

**Reply by ABCDEFG2460** [reply]
+1也是不行

**Comment by diezeit0** [comment]
确实又寄了，一样的提示

**Comment by manmanzhou** [comment]
打开一次deploy.yaml什么都不改再开就能打开了，（玄学😂

**Comment by ChenLily1019** [comment]
将config/deploy.yaml里面的Repository改成下面这个
Repository: 'git://git.lyoko.io/AzurLaneAutoScript'

**Comment by LmeSzinc** [comment]
https://github.com/LmeSzinc/AzurLaneAutoScript/issues/2687#issuecomment-1572526474
